.chloggen/fipsswitch.yaml

```diff
@@ -0,0 +1,12 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+# The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, operator, chart, other)
+component: all
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Introduce a new option `image.otelcol.fips`, when set to true, it sets the appropriate repo/image and skips `migrate-checkpoint` when log is enabled
```

Comment:
Are we officially rolling out our FIPs image? I am asking cause we have previously closed [similarly PRs](https://github.com/signalfx/splunk-otel-collector-chart/pull/1687) and have been waiting for green light from product.

Reply:
I was not aware of this PR, I wonder if this is the same user who reported it internally.
The workaround provided in the linked PR is not ideal as you will loose some functionalities by skipping all init containers.
As for the fips image support, this implies https://github.com/signalfx/splunk-otel-collector-chart/blob/main/helm-charts/splunk-otel-collector/values.yaml#L976 that it's supported.